### DISCUSSION TITLE
Sharing data with group?

### DISCUSSION THREAD

**Discussion by JtheBAB** [baimard/gestion#198] [General]
Hi

I would like to use this great tool. But it looks like the data is exclusive for a user. Of course i could create a special user for that but somehow it would be easier for my people if we could just define that Group A has access to the same data set.

Best wishes

JtheBAB

**Comment by baimard** [comment]
Yes it's a very good idea, and in fact it's in my plan.

I'm going to add this in the roadmap.

Could you explain, how you would see this feature?

**Comment by JtheBAB** [comment]
Perhaps the admin could create a "Company" and assign either Nextcloud Users or Groups to it. Then everyone assigned to the "Company" would see the same data. The question would then be where the pdf invoices are saved.

**Comment by baimard** [comment]
It's a very very very good idea !

I think you can create pdf invoice in a shared folder for the same group.

**Comment by flammableliquids** [comment]
Please prioritize this feature. Having a 'service account' creates a security risk and a user hurdle that is quite frustrating. In the end I have had to limit the clients created to people who are working on them.

I understand that this app was written for micro-companies, but 3 people is enough to make this frustrating on a weekly basis.

**Reply by baimard** [reply]
You can ask for a special request, https://www.buymeacoffee.com/benjaminaimard/commissions

**Reply by flammableliquids** [reply]
Times are tough. I did what I could, and I have minimal expectations.

**Reply by baimard** [reply]
Thank you for sponsoring, I will prioritize this feature

**Comment by baimard** [comment]
this feature is the next.

**Comment by baimard** [comment]
Dear everybody,

I finished with this features ! You can now share with others users !!! in 2.6.1
